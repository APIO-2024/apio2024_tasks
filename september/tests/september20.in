txy4h26c1rm1uv8tr3eonahd67u8h56x
53
10 1
0 1 2 3 4 5 6 7 8
9 8 6 7 2 5 3 1 4
10 1
0 1 2 3 4 5 6 7 8
5 7 9 2 6 4 3 1 8
10 1
0 1 2 3 4 5 6 7 8
9 7 8 6 5 4 3 2 1
10 1
0 1 2 3 4 5 6 7 8
8 9 7 6 5 4 2 3 1
10 1
0 1 2 3 4 5 6 7 8
7 8 6 5 4 1 3 9 2
10 1
0 1 2 3 4 5 6 7 8
8 9 7 6 4 5 3 2 1
10 1
0 1 2 3 4 5 6 7 8
9 6 5 8 7 4 2 3 1
10 1
0 1 2 3 4 5 6 7 8
9 8 5 3 1 4 6 7 2
10 1
0 1 2 3 4 5 6 7 8
9 5 7 8 6 4 2 3 1
10 1
0 1 2 3 4 5 6 7 8
8 3 5 7 9 2 4 1 6
10 1
0 1 2 3 4 5 6 7 8
9 8 7 6 4 5 2 3 1
10 1
0 1 2 3 4 5 6 7 8
9 8 6 7 2 3 1 4 5
10 1
0 1 2 3 4 5 6 7 8
9 8 3 5 2 7 6 4 1
10 1
0 1 2 3 4 5 6 7 8
9 8 7 6 4 5 3 1 2
10 1
0 1 2 3 4 5 6 7 8
9 8 7 6 5 4 3 2 1
10 1
0 1 2 3 4 5 6 7 8
4 6 7 5 3 9 8 1 2
10 1
0 1 2 3 4 5 6 7 8
8 7 9 6 4 5 3 2 1
10 1
0 1 2 3 4 5 6 7 8
7 5 8 6 9 4 3 1 2
10 1
0 1 2 3 4 5 6 7 8
9 8 6 7 5 4 3 2 1
10 1
0 1 2 3 4 5 6 7 8
7 6 8 5 9 4 2 3 1
40 1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38
37 39 32 38 36 33 35 34 31 27 29 28 30 24 17 14 21 26 22 18 25 20 23 19 15 16 13 11 7 2 12 1 4 5 9 3 8 10 6
40 1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38
37 32 33 34 39 38 35 36 31 30 26 29 25 28 24 27 22 20 23 19 21 11 18 15 12 13 17 16 14 10 7 5 1 2 3 8 9 6 4
40 1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38
38 39 37 35 36 34 32 33 31 30 26 29 28 27 24 23 25 21 22 20 17 18 16 19 14 15 12 11 13 10 9 8 7 6 4 3 5 2 1
40 1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38
37 36 39 38 35 33 34 32 31 30 28 29 27 26 23 24 25 22 20 21 18 19 17 16 14 15 11 10 12 13 9 8 7 6 3 2 4 5 1
40 1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38
39 38 37 36 35 34 33 32 31 30 29 28 27 26 25 21 24 22 23 19 20 17 18 15 16 13 14 12 11 10 8 9 7 6 5 3 4 2 1
100 1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 71 72 73 74 75 76 77 78 79 80 81 82 83 84 85 86 87 88 89 90 91 92 93 94 95 96 97 98
99 98 96 97 94 93 95 92 90 91 89 87 88 86 85 84 83 82 80 81 79 78 76 77 75 74 73 72 71 70 69 68 67 66 64 65 62 63 61 60 59 58 57 56 55 54 52 53 50 51 48 49 47 46 44 45 43 42 41 40 38 39 37 36 34 35 32 33 31 30 29 28 27 26 25 23 24 22 20 21 19 17 18 16 15 14 13 11 12 9 10 8 7 6 5 4 3 2 1
1000 1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 71 72 73 74 75 76 77 78 79 80 81 82 83 84 85 86 87 88 89 90 91 92 93 94 95 96 97 98 99 100 101 102 103 104 105 106 107 108 109 110 111 112 113 114 115 116 117 118 119 120 121 122 123 124 125 126 127 128 129 130 131 132 133 134 135 136 137 138 139 140 141 142 143 144 145 146 147 148 149 150 151 152 153 154 155 156 157 158 159 160 161 162 163 164 165 166 167 168 169 170 171 172 173 174 175 176 177 178 179 180 181 182 183 184 185 186 187 188 189 190 191 192 193 194 195 196 197 198 199 200 201 202 203 204 205 206 207 208 209 210 211 212 213 214 215 216 217 218 219 220 221 222 223 224 225 226 227 228 229 230 231 232 233 234 235 236 237 238 239 240 241 242 243 244 245 246 247 248 249 250 251 252 253 254 255 256 257 258 259 260 261 262 263 264 265 266 267 268 269 270 271 272 273 274 275 276 277 278 279 280 281 282 283 284 285 286 287 288 289 290 291 292 293 294 295 296 297 298 299 300 301 302 303 304 305 306 307 308 309 310 311 312 313 314 315 316 317 318 319 320 321 322 323 324 325 326 327 328 329 330 331 332 333 334 335 336 337 338 339 340 341 342 343 344 345 346 347 348 349 350 351 352 353 354 355 356 357 358 359 360 361 362 363 364 365 366 367 368 369 370 371 372 373 374 375 376 377 378 379 380 381 382 383 384 385 386 387 388 389 390 391 392 393 394 395 396 397 398 399 400 401 402 403 404 405 406 407 408 409 410 411 412 413 414 415 416 417 418 419 420 421 422 423 424 425 426 427 428 429 430 431 432 433 434 435 436 437 438 439 440 441 442 443 444 445 446 447 448 449 450 451 452 453 454 455 456 457 458 459 460 461 462 463 464 465 466 467 468 469 470 471 472 473 474 475 476 477 478 479 480 481 482 483 484 485 486 487 488 489 490 491 492 493 494 495 496 497 498 499 500 501 502 503 504 505 506 507 508 509 510 511 512 513 514 515 516 517 518 519 520 521 522 523 524 525 526 527 528 529 530 531 532 533 534 535 536 537 538 539 540 541 542 543 544 545 546 547 548 549 550 551 552 553 554 555 556 557 558 559 560 561 562 563 564 565 566 567 568 569 570 571 572 573 574 575 576 577 578 579 580 581 582 583 584 585 586 587 588 589 590 591 592 593 594 595 596 597 598 599 600 601 602 603 604 605 606 607 608 609 610 611 612 613 614 615 616 617 618 619 620 621 622 623 624 625 626 627 628 629 630 631 632 633 634 635 636 637 638 639 640 641 642 643 644 645 646 647 648 649 650 651 652 653 654 655 656 657 658 659 660 661 662 663 664 665 666 667 668 669 670 671 672 673 674 675 676 677 678 679 680 681 682 683 684 685 686 687 688 689 690 691 692 693 694 695 696 697 698 699 700 701 702 703 704 705 706 707 708 709 710 711 712 713 714 715 716 717 718 719 720 721 722 723 724 725 726 727 728 729 730 731 732 733 734 735 736 737 738 739 740 741 742 743 744 745 746 747 748 749 750 751 752 753 754 755 756 757 758 759 760 761 762 763 764 765 766 767 768 769 770 771 772 773 774 775 776 777 778 779 780 781 782 783 784 785 786 787 788 789 790 791 792 793 794 795 796 797 798 799 800 801 802 803 804 805 806 807 808 809 810 811 812 813 814 815 816 817 818 819 820 821 822 823 824 825 826 827 828 829 830 831 832 833 834 835 836 837 838 839 840 841 842 843 844 845 846 847 848 849 850 851 852 853 854 855 856 857 858 859 860 861 862 863 864 865 866 867 868 869 870 871 872 873 874 875 876 877 878 879 880 881 882 883 884 885 886 887 888 889 890 891 892 893 894 895 896 897 898 899 900 901 902 903 904 905 906 907 908 909 910 911 912 913 914 915 916 917 918 919 920 921 922 923 924 925 926 927 928 929 930 931 932 933 934 935 936 937 938 939 940 941 942 943 944 945 946 947 948 949 950 951 952 953 954 955 956 957 958 959 960 961 962 963 964 965 966 967 968 969 970 971 972 973 974 975 976 977 978 979 980 981 982 983 984 985 986 987 988 989 990 991 992 993 994 995 996 997 998
999 998 997 996 995 994 993 991 992 989 990 988 987 986 985 984 983 982 981 979 980 977 978 976 974 975 973 972 971 970 969 968 967 966 965 964 963 961 962 960 959 958 957 956 955 954 953 952 951 950 949 948 946 947 945 944 943 941 942 940 939 938 937 936 934 935 932 933 931 930 929 928 927 926 925 924 923 922 920 921 919 918 916 917 915 914 913 912 910 911 909 908 907 906 905 904 903 902 901 899 900 898 897 895 896 893 894 892 891 890 889 888 887 886 885 884 883 882 880 881 879 878 877 876 875 874 873 871 872 870 869 867 868 866 864 865 863 862 861 860 859 858 857 856 855 854 853 852 851 850 849 848 847 846 845 844 842 843 841 840 839 838 837 836 835 834 833 832 831 830 828 829 827 826 825 824 823 822 821 820 819 818 817 816 815 814 813 812 811 810 809 807 808 806 804 805 803 802 800 801 798 799 797 796 795 793 794 792 791 790 789 788 787 785 786 784 783 782 781 779 780 778 777 776 774 775 773 772 771 770 769 768 767 766 765 764 763 762 761 760 759 758 757 756 755 754 753 751 752 750 748 749 747 746 745 744 743 742 740 741 739 738 737 736 735 734 732 733 731 730 729 728 727 725 726 724 722 723 721 720 719 718 716 717 715 714 713 712 711 709 710 707 708 705 706 704 703 702 701 700 699 698 697 696 694 695 693 692 691 689 690 687 688 686 685 684 682 683 681 680 679 678 677 675 676 674 672 673 671 670 668 669 667 666 665 664 662 663 661 660 659 658 656 657 655 654 653 651 652 650 648 649 647 646 644 645 643 642 641 640 639 638 637 636 635 633 634 632 631 630 629 627 628 626 625 624 623 622 621 620 618 619 617 615 616 614 613 612 610 611 608 609 607 605 606 604 603 602 600 601 599 598 597 596 595 594 592 593 591 590 589 588 587 585 586 584 583 582 581 580 578 579 577 576 575 573 574 572 570 571 569 568 567 566 565 563 564 562 560 561 559 558 557 556 555 554 553 551 552 550 549 548 547 546 545 544 543 542 541 540 539 538 537 536 535 533 534 532 531 530 529 528 527 526 524 525 523 522 521 520 519 518 517 516 515 514 513 511 512 510 509 508 507 506 505 504 503 502 500 501 499 498 497 495 496 494 492 493 490 491 488 489 487 486 484 485 483 482 481 480 479 478 477 476 474 475 473 471 472 470 469 468 467 466 465 464 463 462 461 460 458 459 456 457 455 454 453 452 451 450 448 449 447 445 446 443 444 442 441 440 439 438 437 436 434 435 433 432 431 430 429 428 427 426 424 425 423 422 421 420 419 417 418 415 416 414 413 412 410 411 409 408 407 406 405 403 402 404 401 400 399 398 397 396 395 394 393 391 392 389 390 388 387 386 385 384 383 381 382 380 379 378 377 376 375 374 373 371 372 370 369 367 368 366 365 363 364 362 361 360 359 357 358 356 354 355 353 352 351 350 349 347 348 346 345 343 344 342 341 340 339 338 337 336 335 334 333 331 332 330 329 328 326 327 325 324 323 321 322 319 320 318 317 316 315 314 313 312 311 310 309 308 307 306 304 305 303 302 301 300 299 298 297 295 296 294 293 291 292 290 289 287 288 286 285 284 282 283 281 280 279 278 277 276 275 273 274 272 271 270 269 268 267 266 265 264 263 262 261 260 259 258 257 256 254 255 253 252 251 249 250 247 248 245 246 244 242 243 240 241 239 238 237 236 235 234 233 232 231 230 228 229 227 225 226 224 222 223 221 220 219 218 217 216 215 214 213 211 212 210 209 207 208 206 205 204 203 202 201 200 199 197 198 196 195 194 193 192 191 190 188 189 186 187 185 184 183 182 181 180 179 177 178 175 176 174 173 172 171 170 169 168 167 166 165 164 163 162 161 160 158 159 157 156 155 154 153 151 152 150 149 148 146 147 145 144 143 142 141 140 139 138 136 137 135 134 133 132 131 129 130 128 126 127 125 124 123 122 121 120 118 119 117 116 115 113 114 112 111 110 109 107 108 106 105 103 104 101 102 100 99 98 97 96 94 95 93 92 91 90 89 88 87 86 85 83 84 82 81 80 78 79 76 77 75 74 73 72 71 69 70 68 67 66 65 64 63 62 61 60 59 58 57 56 55 54 53 52 51 49 50 47 48 46 45 43 44 42 41 40 39 38 36 37 35 34 33 32 31 30 29 28 27 26 25 24 23 22 21 20 19 18 17 16 15 14 13 12 11 10 9 8 7 6 5 4 3 2 1
100 1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 71 72 73 74 75 76 77 78 79 80 81 82 83 84 85 86 87 88 89 90 91 92 93 94 95 96 97 98
99 97 98 96 95 93 94 92 91 90 87 88 89 85 86 84 83 82 81 79 80 77 76 78 75 74 73 72 70 71 69 68 67 66 65 64 63 62 61 60 59 57 56 58 55 53 54 52 51 50 49 48 46 47 45 44 43 42 41 40 37 38 39 36 35 34 33 30 32 31 29 27 28 26 25 24 23 22 21 19 20 18 16 17 15 14 13 12 11 10 9 8 7 6 5 3 4 2 1
40 1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38
39 37 38 36 34 35 33 32 31 29 30 28 27 26 25 23 24 21 22 20 19 18 17 16 15 14 13 11 10 12 9 8 7 6 5 4 3 2 1
40 1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38
38 39 37 36 35 34 32 33 31 29 30 28 27 25 26 24 23 22 21 19 18 20 17 15 16 14 13 12 10 11 9 8 6 7 4 5 3 2 1
40 1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38
38 39 37 35 36 34 33 32 31 30 29 27 26 28 25 24 21 23 22 20 19 18 16 15 17 14 13 11 12 10 9 8 7 6 5 3 4 1 2
40 1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38
39 37 32 34 33 36 35 38 29 28 31 30 27 26 20 22 24 25 19 23 21 18 15 12 17 13 14 11 16 10 9 8 7 3 5 4 1 2 6
40 1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38
39 38 36 37 35 34 33 31 32 29 30 28 27 26 25 24 23 22 21 20 19 18 17 16 15 14 12 13 11 10 9 8 7 6 5 4 2 3 1
10 1
0 1 2 3 4 5 6 7 8
9 7 8 6 4 5 3 2 1
10 1
0 1 2 3 4 5 6 7 8
9 7 8 6 5 4 3 2 1
10 1
0 1 2 3 4 5 6 7 8
9 8 7 6 4 5 2 3 1
10 1
0 1 2 3 4 5 6 7 8
9 8 7 6 5 3 4 1 2
10 1
0 1 2 3 4 5 6 7 8
9 8 6 7 5 4 3 2 1
10 1
0 1 2 3 4 5 6 7 8
6 3 8 5 4 9 7 2 1
10 1
0 1 2 3 4 5 6 7 8
9 8 5 6 4 2 1 3 7
10 1
0 1 2 3 4 5 6 7 8
8 9 7 6 4 5 3 2 1
10 1
0 1 2 3 4 5 6 7 8
9 8 7 6 5 4 3 2 1
10 1
0 1 2 3 4 5 6 7 8
8 9 7 5 6 4 3 2 1
10 1
0 1 2 3 4 5 6 7 8
5 7 8 6 9 4 3 2 1
10 1
0 1 2 3 4 5 6 7 8
9 8 7 6 5 4 3 2 1
10 1
0 1 2 3 4 5 6 7 8
9 7 8 6 5 4 3 2 1
10 1
0 1 2 3 4 5 6 7 8
8 9 7 6 5 3 4 1 2
10 1
0 1 2 3 4 5 6 7 8
9 8 7 6 5 3 4 2 1
10 1
0 1 2 3 4 5 6 7 8
8 9 7 6 5 3 4 2 1
10 1
0 1 2 3 4 5 6 7 8
9 8 7 6 5 4 3 2 1
10 1
0 1 2 3 4 5 6 7 8
9 8 7 6 4 5 3 2 1
10 1
0 1 2 3 4 5 6 7 8
9 8 7 6 5 4 3 1 2
10 1
0 1 2 3 4 5 6 7 8
9 8 7 6 5 4 3 2 1
