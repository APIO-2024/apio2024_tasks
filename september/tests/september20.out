7ckgnn4wyi495puj3ibqf81dqvapyv6b
4
1
8
7
1
7
5
3
5
1
7
4
4
7
9
2
6
4
8
4
6
8
22
23
30
78
852
78
32
29
28
13
34
7
8
7
7
8
3
3
7
9
7
5
9
8
6
8
7
9
8
8
9
